### PR TITLE
Adding routes

### PR DESCRIPTION
**Pull request by reecemillsom**
**TESTING**

* Pull branch down, make sure tests run and are outputting correctly, ensure that when you change the assertion in the tests the tests fail or pass depending on what you try.

* When you pull down the branch run ```rm -rf node_modules/ && npm it```

* Check that all dependencies we have installed are being used if not comment on here and we can look together.

* Make sure that npm start still works and that navigating to a route still works, wont return anything though due to nothing in the one route we have. Paste after npm start in browser ```localhost:3000/```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "chai": "^4.1.1",
+    "dirty-chai": "^2.0.1",
+    "karma": "^1.7.0",
+    "karma-babel-preprocessor": "^6.0.1",
```
Comment (reecemillsom) [inline package.json]: Not sure if this is still being used

File: test/mocha.opts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+--colors
+--reporter progress
```
Comment (reecemillsom) [inline test/mocha.opts]: May be able to remove this

Comment (SimonKaya91) [reply test/mocha.opts]: Removing reporter line tests still do still pass

File: package.json (hunk 1 above)
Comment (SimonKaya91) [reply package.json]: At first glance i would say yes as the Karma config file has it as a plugin and uses babel as a pre processor on the spec file path. Commenting out the plugin of the karma file we get a load error. Are you on about getting rid of babel pre processor as well with it?

Comment (reecemillsom) [reply package.json]: Idea being karma-babel-preprocessor is removed from the config and as a node modules and seeing if everything still works without it as technically I don't think we are using if it doesn't work without it just checkout changed

Comment (SimonKaya91) [reply package.json]: Taking babel out from the config file totally still completes the tests so we could remove, but I thought we needed babel for compiling down to regular js later on project?

Comment (reecemillsom) [reply package.json]: Babel yeah but not sure karma-babel-preprocessor as not sure it's doing anything within config file, when removed it run npm I remove from config file and try run the tests

**Pull request merged**